contracts/test/DelegateCaller.sol

```diff
@@ -13,12 +13,15 @@ contract DelegateCaller {
     function makeDelegatecall(address _called, bytes memory _calldata) external returns (bool success, bytes memory returnData) {
         (success, returnData) = _called.delegatecall(_calldata);
         if (!success) {
-            // solhint-disable-next-line no-inline-assembly
+            /* solhint-disable no-inline-assembly */
+            /// @solidity memory-safe-assembly
             assembly {
+                let ptr := mload(0x40)
```

Comment:
Same question here: do we want to ensure memory safety to non-production contracts as the original issue was about fixing the compilation, and those do not affect it?

I'm not against it, I want to ensure we're all on the same page.

Reply:
Honestly, I am not sure. I was considering all the contracts (except the test/mock ones) for this task just to be on the safe side. Maybe Nick can answer this one better.

Reply:
Yes, I agree. We can run it by him once he's back next week.

Reply:
Resolved 👍🏾